feedback/migrations/0020_remove_feedbacktag_feedbacks_conversation_and_more.py

```diff
@@ -4,17 +4,46 @@
 import django.db.models.deletion
 
 
+def generate_conversations_for_old_feedback(apps, schema_editor):
+    Conversation = apps.get_model("feedback", "Conversation")
+    Feedback = apps.get_model("feedback", "Feedback")
+    for fb in Feedback.objects.all():
+        conv, _created = Conversation.objects.get_or_create(
+            exercise=fb.exercise,
+            student=fb.student,
+        )
+        fb.conversation = conv
+        fb.save()
+
+
+def delete_conversations(apps, schema_editor):
+    Conversation = apps.get_model("feedback", "Conversation")
+    for conv in Conversation.objects.all():
+        conv.delete()
```

Comment:
I believe you can just do `Conversation.objects.all().delete()` which would be much faster in the unlikely event that the migration needs to be reversed.